src/Constants.jsx

```diff
@@ -0,0 +1,6 @@
+export const Constants = {
+  readMore: '... Read More',
+  showLess: 'Show Less',
+  // max chars to be shown in collapsed view of the message
+  maxChars: 130,
+};
```

Comment:
Always leave a newline at end of file, it's a good practice. You can read more about that [here](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.)

Reply:
Sure. May I use it from next time ?(PR) as it's conflicting with few other branches. @majordwarf

Reply:
not that much additional work to just fix the change then and there.